rs/nns/sns-wasm/canister/canister.rs

```diff
@@ -334,6 +335,12 @@ fn canister_post_upgrade() {
     println!("{}Completed post upgrade", LOG_PREFIX);
 }
 
+// TODO[NNS1-2997]: Remove this function once the migration is completed.
+#[heartbeat]
```

Comment:
Let's use the dfn_core version of heartbeat until we migrate the canister to ic_cdk?